### ISSUE TITLE
pfx2john produces invalid files

### ISSUE THREAD

**Issue by gjvnq** [openwall/john#4465]
### Steps to reproduce ###
1. Run `pfx2john user.pfx > user.john`
2. Run `john user.john`
3. Get error: No password hashes loaded (see FAQ)

The `user.john` file looks like:
```
user.pfx:$pfxng$1$a0$1$8$b'053f66873a74d1b5'$b'0f820[REDACTED LONG HEX SEQUENCE]9f3':::::user.pfx
```

After some trying and error, I noticed that the examples in the test file (`pfx_fmt_plug.c`) didn't have these `b'` sequences. After removing them and the corresponding `'`, the program worked.

I think this is a Python 2 vs Python 3 bug.

**Comment by solardiz** [comment]
@gjvnq What version of JtR are you using? Since you demonstrate a good understanding of related matters above, I naively assumed the latest and have just wasted some time on this, only to conclude that you were likely _not_ using the latest, as we seem to have already fixed this issue in #4093. Please confirm that we can indeed close this issue as it's already fixed, or correct me if not. Thanks.

**Comment by gjvnq** [comment]
John the Ripper 1.9.0-jumbo-1 MPI + OMP [linux-gnu 64-bit x86_64 AVX AC]

I just checked my /usr/bin/pfx2john and it does not contain the modifications made on #4093. I.e. I thought I was running the latest version but I wasn't.

I assumed JtR used semantic versioning but I was wrong.

**Comment by vaknin** [comment]
I am still experiencing this issue, yet it was supposed to be fixed in [#4093](https://github.com/openwall/john/pull/4093).
Just like @solardiz, I am running JtR 1.9.0, i.e. the latest version available.

**Comment by ghost** [comment]
> I am still experiencing this issue, yet it was supposed to be fixed in [#4093](https://github.com/openwall/john/pull/4093). Just like @solardiz, I am running JtR 1.9.0, i.e. the latest version available.

This is because you are using the latest stable (older) version. Please, download and use the version from this repository. It contains the fix [#4093].
